scripts/cpp_generator.py

```diff
@@ -201,12 +201,10 @@ def __init__(self, cmd, gen):
         self.returns = ['result']
         self.return_template_params = []
 
-        self.dispatch = "Dispatch&& d"
-        self.suppress_default_dispatch_arg = not self.is_core
-        self.template_decl_list = ["typename Dispatch"]
-        self.template_defn_list = self.template_decl_list[:]
-        if self.is_core:
-            self.template_decl_list[0] = self.template_decl_list[0] + " = DispatchLoaderStatic"
+        self.dispatch = "Dispatch d"
+        self.suppress_default_dispatch_arg = False
```

Comment:
So the purpose of "suppress" here is that the "default dispatcher" (which was the static one) isn't a possibility for extension-defined functions.  Maybe we can do a default core dispatcher and a default extension dispatcher, and only add the ` = decltype((OPENXR_HPP_DEFAULT_DISPATCHER))` if the `OPENXR_HPP_DEFAULT_DISPATCHER` extension dispatcher is defined?  We'd need to do something similar below too.